src/tests/ftest/util/server_utils.py

```diff
@@ -908,6 +908,75 @@ def kill(self):
         # set stopped servers state to make teardown happy
         self.update_expected_states(None, ["stopped", "excluded", "errored"])
 
+    @fail_on(CommandFailure)
+    def system_exclude(self, ranks, daos_log, copy=False, rank_hosts=None):
+        """Exclude the specific server ranks.
+
+        Args:
+            ranks (list): a list of daos server ranks (int) to exclude
+            daos_log (DaosLog): object for logging messages
+            copy (bool, optional): Copy dmg command. Defaults to False.
+            rank_hosts (str): hostlist representing hosts whose managed ranks are to be
+                operated on.
+
+        Raises:
+            avocado.core.exceptions.TestFail: if there is an issue excluding the server
+                ranks.
+
+        """
+        msg = "Excluding DAOS ranks {} from server group {}".format(
+            ranks, self.get_config_value("name"))
+        self.log.info(msg)
+        daos_log.info(msg)
```

Comment:
I wonder if this really needed.  The idea behind it was to log messages in a `DaosLog` object to mark test events in a daos log, but we really don't use it that much.  @daltonbohning thoughts?

Reply:
Removed.